examples/tracing-compare.cpp

```diff
@@ -0,0 +1,178 @@
+/**
```

Comment:
Why are we adding another utility for running resnet?  What if I want to profile german-to-english or AlexNet?

Reply:
Overall, half of the general functionality in this file seems to be very similar to ModelRunner/Loader/ImageClassifier. Just recently I've written something similar for one of the backends and also had to duplicate a lot of code.   

Ideally, we should re-factor  ModelRunner/Loader/ImageClassifier and make this functionality re-usable. This way we could easily write small tests/tools like this without duplicating a lot of boiler-plate code.

I filed https://github.com/pytorch/glow/issues/2381 to track this.

Reply:
Don't object to adding this to the image classifier tool at all, but I was making a distinction here between tools (for running and getting results) and examples (code-based documentation). 

Definitely like the idea of pulling out some common code here.